src/braket/aws/aws_device.py

```diff
@@ -622,3 +666,133 @@ def get_device_region(device_arn: str) -> str:
                 f"Device ARN is not a valid format: {device_arn}. For valid Braket ARNs, "
                 "see 'https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html'"
             )
+
+    def refresh_gate_calibrations(self) -> GateCalibrations:
```

Comment:
The refresh will attempt to pull the latest data available. If there is no latter data between refresh calls, then it will stay the same. The URL requested directs to the latest data as it removes the query params.

We can add a refresh_metadata call in here to keep the two in sync.